plugins/metrics/runtime/runtime.go

```diff
@@ -0,0 +1,89 @@
+package runtime
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/otel/api/metric"
+	"go.opentelemetry.io/otel/api/unit"
+	"golang.org/x/time/rate"
+
+	"github.com/open-telemetry/opentelemetry-go-contrib/plugins/metrics/runtime/internal/metrics"
+)
+
+var (
+	initialisation time.Time
+)
+
+func init() {
+	initialisation = time.Now()
+}
+
+// various runtime metrics have a certain overhead, and as such, we only measure them on a defined interval
+func Runtime(ctx context.Context, meter metric.Meter, interval time.Duration) context.CancelFunc {
+	ctx, cancel := context.WithCancel(ctx)
+
+	ctxTimeout, _ := context.WithTimeout(ctx, time.Second)
+	var m, p metrics.Metrics
+	m, _ = metrics.Measure(ctxTimeout)
+
+	if interval == 0 {
+		interval = time.Second
+	}
+
+	go func() {
+		limit := rate.NewLimiter(rate.Every(time.Second), 1)
+		for {
+			err := limit.Wait(ctx)
+
+			if err != nil {
+				break
+			}
+
+			ctxTimeout, _ := context.WithTimeout(ctx, time.Second)
+			// store the previous measurement for reporting deltas
+			p = m
+			// we assume here we are replacing m and thus do ont require a concurrency lock
+			m, _ = metrics.Measure(ctxTimeout)
+
+			metric.Must(meter).NewFloat64Counter("runtime.go.cpu.user").Bind().Add(ctx,
+				m.ProcessCPU.User-p.ProcessCPU.User,
+			)
+
+			metric.Must(meter).NewFloat64Counter("runtime.go.cpu.sys").Bind().Add(ctx,
```

Comment:
OTEPs 93/96 introduce a new metric instrument that will let us Observe a cumulative and not need to compute a delta.  This is fine since we don't have either the batch-observer or cumulative instrument here, yet. This will become a CumulativeObserver, deltas will be computed in the SDK because the exporter (in your case) will demand it.